src/object.d

```diff
@@ -1526,41 +1526,73 @@ const:
 
     @property uint flags() nothrow pure @nogc { return _flags; }
 
+    /************************
+     * Returns:
+     *  module constructor for thread locals, `null` if there isn't one
```

Comment:
There is no convention. In any case, non-sentences shouldn't be capitalized nor end in periods.